microprofile/microprofile-opentracing-1.3/src/main/java/org/wildfly/swarm/ts/microprofile/opentracing/v13/Resource.java

```diff
@@ -0,0 +1,67 @@
+package org.wildfly.swarm.ts.microprofile.opentracing.v13;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.core.Response;
+
+@Path("/")
+public class Resource {
+    @Inject
+    private TracedService service;
+
+    @GET
+    @Path("/tracedlogged")
+    public Response tracedLoggedMethod() {
+        return Response.ok().entity(service.tracedLoggedMethod()).build();
+    }
+
+    @GET
+    @Path("/traced")
+    public Response tracedMethod() {
+        return Response.ok().entity(service.tracedMethod()).build();
+    }
+
+    @GET
+    @Path("/logged")
+    public Response loggedMethod() {
+        return Response.ok().entity(service.loggedMethod()).build();
+    }
+
+    @GET
+    @Path("/nottracednotlogged")
+    public Response notTracedNotLoggedMethod() {
+        return Response.ok().entity(service.notTracedNotLoggedMethod()).build();
+    }
+
+    @GET
+    @Path("/named/true")
+    public Response tracedNamedTrue() {
+        return Response.ok().entity(service.tracedNamedTrue()).build();
+    }
+
+    @GET
+    @Path("/true")
+    public Response tracedTrue() {
+        return Response.ok().entity(service.tracedTrue()).build();
+    }
+
+    @GET
+    @Path("/named/false")
+    public Response tracedNamedFalse() {
+        return Response.ok().entity(service.tracedNamedFalse()).build();
+    }
+
+    @GET
+    @Path("/false")
+    public Response tracedFalse() {
+        return Response.ok().entity(service.tracedFalse()).build();
+    }
+
+    @GET
+    @Path("/test/{id: \\d+}/{txt: \\w+}")
+    public Response twoWildcard(@PathParam("id") long id, @PathParam("txt") String txt) {
```

Comment:
I'd still like to understand what does this method add that isn't tested by the previous methods. Are the parameters traced automatically or something like that?

Reply:
This method adds nothing related to tracing. It's just confirmation that "Clarify http-path when path contains regular expressions (#136 https://github.com/eclipse/microprofile-opentracing/pull/136)" works.

Reply:
Cool, thanks! If a link to that issue was in a comment above this method (or somewhere else), I wouldn't have to ask :-)